cmake/Modules/Findbabl.cmake

```diff
@@ -0,0 +1,48 @@
+set(PKG_CONFIG_USE_CMAKE_PREFIX_PATH ON)
+find_package(PkgConfig)
+pkg_check_modules(PC_BABL QUIET babl)
+
+set(babl_VERSION ${PC_BABL_VERSION})
+
+find_path(babl_INCLUDE_DIR
+  NAMES babl/babl.h
+  HINTS
+    ${babl_DIR}/include
+    ${PC_BABL_INCLUDE_DIRS}
+  DOC "babl include dir"
```

Comment:
I would add a `PATH_SUFFIXES babl-0.1` here, since at least on Fedora the main header is installed to `/usr/include/babl-0.1/babl/babl.h`. In a non-pkgconfig situation where the `HINTS` aren't matched, the automatic `/usr/include/` scanning should find the header if it knows to also check using that suffix.

...Hrm, and now I'm curious why the Fedora 34 `babl-0.1.86` package contains a header with these defines:
```c++
#define BABL_MAJOR_VERSION 0
#define BABL_MINOR_VERSION 1
#define BABL_MICRO_VERSION 86
```
...but a library file named `/usr/lib64/libbabl-0.1.so.0.185.1`. :confused: